ROADMAP.md

```diff
@@ -22,9 +22,21 @@ rkt's version 1.0 release marks the command line user interface and on-disk data
   - Fedora [#1304](https://github.com/coreos/rkt/issues/1304)
   - Debian [#1307](https://github.com/coreos/rkt/issues/1307)
 
-### rkt 1.1 (Future)
+### rkt 1.1 (February)
 
-- stable API
+- Enhanced DNS configuration [#2044](https://github.com/coreos/rkt/issues/2044)
+- User configuraion for stage1 [#2013](https://github.com/coreos/rkt/issues/2013)
+
+### rkt 1.2 (March)
+
+- app exit status propagation [#1460](https://github.com/coreos/rkt/issues/1460)
+- `rkt fly` as top-level command [#1889](https://github.com/coreos/rkt/issues/1889)
 - fully integrated with `machinectl login` and `systemd-run` [#1463](https://github.com/coreos/rkt/issues/1463)
 - IPv6 support [appc/cni#31](https://github.com/appc/cni/issues/31)
 - attach to the app's stdin/stdout [#1799](https://github.com/coreos/rkt/issues/1799)
+
+### rkt 1.3 (March)
+
+- stable API
+- full integration with Kubernetes (aka "rktnetes")
+- full integration with `machinectl login` and `systemd-run` [#1463](https://github.com/coreos/rkt/issues/1463)
```

Comment:
I would add support for unified cgroup hierarchy: https://github.com/coreos/rkt/issues/1757